assets/erc-7844/CDSMinimal.sol

```diff
@@ -0,0 +1,314 @@
+//SPDX-License-Identifier: Unlicensed
```

Comment:
You'll need to license this under something more permissive. CC0-1.0 is preferred, but MIT/Apache would be okay too.